syntax = "proto3";

package transaction;

// Message for TransactionModel
message TransactionModelProto {
    string stripeId = 1;
    string amount = 2;
    string order_id = 3;
}

// Message for Transaction
message TransactionProto {
    TransactionModelProto TransactionModel = 1;
    int32 user_id = 2;
}
